### PR TITLE
Added ENF2.0 and CWA 1.9 description to the FAQ

### PR DESCRIPTION
**Pull request by mtb77**

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -570,6 +570,44 @@
...
+                            "<ul><li>Gewichtete Kontaktzeit <15 Minuten: Begegnung mit niedrigem Risiko (grün)</li>",
+                            "<li>Gewichtete Kontaktzeit >=15 Minuten: Begegnung mit erhöhtem Risiko (rot)</li></ul>",
+                            "Abschließend werden alle Zeitfenster eines Tages betrachtet (die gewichteten Kontaktzeiten) summiert. Ist deren Summe >= 15 Minuten, wird der Gesamtstatus rot, ansonsten ist dieser grün. Letztlich bedeutet das: Mehrere „grüne“ Begegnungen (Zeitfenster) können in Summe zu einem roten Status führen. Wer noch tiefer in die Materie einsteigen möchte: ",
+                            "Es stehen die aktualisierten Diagramme auf Github: <a href='https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation'>https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation</a>",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:607]: Could you update this sentence how it has been done in #700?

**Review by Ein-Tim — CHANGES_REQUESTED** [review]

File: src/data/faq_de.json (hunk 1 above)
Comment (mtb77) [reply src/data/faq_de.json:607]: What do you mean - which change?

Comment (mtb77) [reply src/data/faq_de.json:607]: @Ein-Tim because of the timing i need to merge this first - can you make a second PR please? Thank you!

Comment (Ein-Tim) [reply src/data/faq_de.json:607]:
```suggestion
                            "Die aktualisierten Diagramme stehen auf Github: <a href='https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation'>https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation</a> zur Verfügung",
```

**Comment by Ein-Tim** [comment]
Thanks!

**Review by Ein-Tim — APPROVED** [review]

**Pull request merged**